packages/cli/src/builders.ts

```diff
@@ -159,6 +160,14 @@ const builders: SnapsCliBuilders = {
     demandOption: false,
     default: true,
   },
+
+  typescript: {
+    alias: 'ts',
+    describe: 'Use TypeScript template',
+    type: 'boolean',
+    demandOption: false,
+    default: false,
```

Comment:
If I set that `default` to be `true` then I'll lose the possibility to generate the JavaScript template. It will automatically generate the TypeScript one. So, there will be no use of the --typescript flag anymore (it will always be present in args as `true` by default).

```
mm-snap init => TypeScript
mm-snap init --typescript => TypeScript
```

Reply:
Following the discussion we had and Frederik's suggestion, I will enable TypeScript by default.
So the following commands will generate:
```
mm-snap init => TypeScript
mm-snap init --typescript => TypeScript
mm-snap init --typescript false => JavaScript
```